GFS_layer/GFS_typedefs.F90

```diff
@@ -2052,7 +2108,11 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     logical              :: fixed_sollat   = .false.         !< flag to fix solar latitude
     logical              :: daily_mean     = .false.         !< flag to replace cosz with daily mean value
 
+    !--- dynamical core parameters
+    logical              :: dycore_hydrostatic  = .true.     !< whether the dynamical core is hydrostatic
```

Comment:
The designed way to get this information to the physics from outside is to use the GFS_init_type.  This type is populated with dycore and other external to the physics information within the atmos_model.F90::atmos_model_init function and it is passed to GFS_Initialize, where it is passed to the Model%init procedure.

Reply:
Hi Rusty, thanks for your guidance. I can move these variables to GFS_init_type. I will submit another merge request when it is done.